airflow-core/src/airflow/ui/src/i18n/locales/pl/components.json

```
@@ -10,11 +10,22 @@
     "errorStartDateBeforeEndDate": "Data początkowa musi być wcześniejsza niż data końcowa",
     "maxRuns": "Maksymalna liczba aktywnych wykonań",
     "reprocessBehavior": "Zachowanie ponownego przetwarzania",
-    "run": "Uruchom Backfill",
+    "run": "Uruchom ponowne przetwarzanie",
     "selectDescription": "Uruchom ten DAG dla zakresu dat",
     "selectLabel": "Wypełnienie wsteczne",
     "title": "Uruchom uzupełnienie wsteczne",
-    "unpause": "Wznów {{dag_display_name}} przy wykonaniu"
+    "toaster": {
+      "success": {
+        "description": "Zadania ponownego przetwarzania zostały pomyślnie uruchomione.",
+        "title": "Ponowne przetwarzanie wygenerowany"
```

```suggestion
        "description": "Wykonania ponownego przetwarzania zostały pomyślnie uruchomione.",
        "title": "Wykonanie ponowne przetwarzanie wygenerowane"
```
Maybe something like this? I feel like "jobs" should not be translated as "Zadania", but also "Wykonania" are "Runs", so I'm really confused now.

---

Again -> future "uruchomienie", past -> wykonanie.

---

I updated it to be better.